runtime/src/accounts_db.rs

```diff
@@ -782,30 +782,30 @@ impl AccountsDB {
 
     /// Store the account update.
     pub fn store(&self, fork_id: Fork, accounts: &HashMap<&Pubkey, &Account>) {
-        let mut store_accounts = Measure::start("store::store_accounts");
+        //let mut store_accounts = Measure::start("store::store_accounts");
```

Comment:
did you leave these commented out on purpose?  why not just remove them?

Reply:
rebase mess-up, will remove.